.github/workflows/hlwm.yml

```diff
@@ -210,3 +210,38 @@ jobs:
           else
             echo "Skipping comment posting because of absence of secret"
           fi
+
+  build-doc:
+    name: Build Documentation and website
+    runs-on: ubuntu-latest
+    env:
+      BUILD: build-${{ github.run_number }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: install asciidoc
+        run: sudo apt-get install --no-install-recommends asciidoc xsltproc docbook-xsl
+      - name: restrict cmake to doc
+        run: |
+          cat > CMakeLists.txt <<EOF
```

Comment:
Why is this necessary? Because we cannot do a "docs-only build"?

Reply:
The default ubuntu-latest of github actions does not have the X11-libraries installed and so the `cmake` fails. But I've one had the x libraries installed, `cmake` succeeds and `make all_doc` would only build the doc.

Reply:
Makes sense. So as soon as the dependencies are in the Dockerfile we could just use a regular build and do a all_doc there, sometime in the future.